website/source/docs/guides/creating-certificates.html.md

```diff
@@ -192,7 +267,149 @@ $ export CONSUL_CLIENT_KEY=cli-key.pem
 After these environment variables are correctly configured, the CLI will
 respond as expected.
 
-[cfssl]: https://cfssl.org/
+### Note on SANs for Server and Client Certificates
+
+Using `localhost` and `127.0.0.1` as `Subject Alternative Names` in server
+and client certificates allows tools like `curl` to be able to communicate with
+Consul's HTTPS API when run on the same host. Other SANs may be added during
+server/client certificates creation with `-additional-dnsname` to allow remote
+HTTPS requests from other hosts.
+
+## Configuring the Consul UI for HTTPS
+
+If your servers and clients are configured now like above, you won't be able to
+access the builtin UI anymore. We recommend that you pick a Consul agent you
+want to run the UI on and follow the instructions to get the UI up and running
```

Comment:
Maybe "pick one (or two for availability)"

Reply:
✔️